### PR TITLE
Dynamic user profile display - Ruby's branch

### PR DESCRIPTION
**Pull request by 1rubiz**
Dynamic user profile display

### CODE REVIEW

File: vite.config.ts.timestamp-1714794613150-7e0a1687386af.mjs
Comment (eugenex-lab) [inline vite.config.ts.timestamp-1714794613150-7e0a1687386af.mjs:1]: whats this file ?

Comment (eugenex-lab) [reply vite.config.ts.timestamp-1714794613150-7e0a1687386af.mjs:1]: rm it ?

**Review by eugenex-lab — CHANGES_REQUESTED** [review]
when the user logs out u should clear the local storage user

File: vite.config.ts.timestamp-1714794613150-7e0a1687386af.mjs
Comment (1rubiz) [reply vite.config.ts.timestamp-1714794613150-7e0a1687386af.mjs:1]: It was automatically created when i ran 'npm run dev', however i have removed it and i will take note of it if it comes up again.

**Pull request merged**